src/app/navbar/navbar.component.html

```diff
@@ -14,31 +14,31 @@
         <!--<li><a routerLink="/advertisements" [routerLinkActive]="'active'" [routerLinkActiveOptions]="{ exact: true }">Advertisements</a></li>-->
         <!--<li><a routerLink="/searchAdvertisements" [routerLinkActive]="'active'" [routerLinkActiveOptions]="{ exact: true }"> Search Advertisements</a></li>-->
         <!-- <li><a href="#contact">Contact</a></li> // move to footer -->
-        <app-search-ad></app-search-ad>
+        <!--<app-search-ad></app-search-ad>-->
```

Comment:
Thank you for deleting my search. :rage:

if you don't mind It's better not to touch the other part in order to solve your problems. :tractor: